src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Deprecation.php

```diff
@@ -169,6 +166,26 @@ public function isLegacy($utilPrefix)
             || \in_array('legacy', $test::getGroups($class, $method), true);
     }
 
+    /**
+     * @return bool
+     */
+    public function isMuted()
+    {
+        if (!\in_array($this->message, [
```

Comment:
let's do a plain string comparison (on one line), no need to plan for an hypothetic future.